python/tests/test_misc.py

```diff
@@ -80,4 +80,9 @@ def test_cmake_compilation(sbml_example_presimulation_module):
 
     cmd = f"set -e; cd {source_dir}; mkdir -p build; cd build; "\
           "cmake ..; make"
-    subprocess.run(cmd, shell=True, capture_output=True, check=True)
+
+    # TODO: when tests run on ubuntu20:
```

Comment:
should be done here or later PR?

Reply:
Would like to do separately. A number of things still to be improved, but would first like to see the basic version in action.